tracing/src/span.rs

```diff
@@ -612,7 +590,6 @@ impl Span {
     /// ```
     /// # #[macro_use] extern crate tracing;
     /// # use tracing::Level;
-    /// # fn main() {
```

Comment:
It looks like this doctest no longer compiles (which is [breaking the build](https://github.com/tokio-rs/tracing/pull/491/checks?check_run_id=361598157#step:5:673)) — the `fn main()` is necessary when there's an `extern crate`. Can we remove the `extern crate` here and fix this doctest?